sormas-e2e-tests/src/test/java/org/sormas/e2etests/helpers/WebDriverHelpers.java

```diff
@@ -380,6 +381,41 @@ public String getCheckedOptionFromHorizontalOptionGroup(By options) {
     return baseSteps.getDriver().findElement(options).findElement(SELECTED_RADIO_BUTTON).getText();
   }
 
+  public String getOptionFromHorizontalGroupOrNullIfNothingChecked(By options) {
+    waitUntilIdentifiedElementIsPresent(options);
+    if (baseSteps.getDriver().findElement(options).findElements(SELECTED_RADIO_BUTTON).size()
+        == 0) {
+      return null;
+    } else {
+      return getCheckedOptionFromHorizontalOptionGroup(options);
+    }
+  }
+
+  public boolean isSingleCssInputChecked(By cssInput) {
+    waitUntilIdentifiedElementIsPresent(cssInput);
+    scrollToElement(cssInput);
+    try {
+      boolean value =
+          (baseSteps.getDriver().findElement(cssInput).getAttribute("checked").equals("true"))
+              ? true
+              : false;
+      return value;
+    } catch (Exception e) {
+      return false;
+    }
+  }
+
+  public String getValueForSingleCssInputIfChecked(By cssInput) {
+    if (isSingleCssInputChecked(cssInput)) {
+      String checkedElementCssSelector =
+          StringUtils.getMatchedGroupByIndexFromAString(
+              cssInput.toString(), "(.*cssSelector:\\s)(.*)(\\s>\\sinput.*)", 2);
+      return baseSteps.getDriver().findElement(By.cssSelector(checkedElementCssSelector)).getText();
+    } else {
+      return null;
```

Comment:
@BojteTamas  as you mentioned, to try to avoid returning null. Should be changed with an empty string?